src/v2/Apps/Fair/routes.tsx

```diff
@@ -18,5 +20,35 @@ export const routes: RouteConfig[] = [
         }
       }
     `,
+    children: [
+      {
+        path: "/",
+        getComponent: () => FairOverviewRoute,
+        prepare: () => {
+          FairOverviewRoute.preload()
+        },
```

Comment:
Note that if these sub-routes don't contain a significant amount of stuff that would potentially increase bundle size (libs, etc) they don't _need_ to be bundle split. Only the top level entry. (The UX is slightly snappier without a split.)